### PR TITLE
REF: Privacy to hooks

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/egffjsq72c46ghqlphmetzsxee?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/520)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/53wjlnebd65oawzpkgrzirzobq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/537)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kwsbxaaakycbnalnovnbrrqxvy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/539)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: hooks/usePrivacy.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const usePrivacy = () => {
+  const enableBlur = () => {};
+
+  const disableBlur = () => {};
+
```
Comment (Overtorment) [inline hooks/usePrivacy.tsx:5]: looks funny kek.

**Review by Overtorment — APPROVED** [review]
man it annoys me a lot that there are different packages for blur on ios and android, and both of them are abandoned.

i did a quick googling, is smth like https://www.npmjs.com/package/@react-native-community/blur  an option?

even more, can we implement blur ourselves? 

the way I see it, it could be a component in `index.js` that wraps our  `<App />`, smth like:


```js
    <BlueStorageProvider>
      <BluePrivacy>
        <App />
     </BluePrivacy>
    </BlueStorageProvider>
```

or even simpler, a conditional (with a condition supplied by context) that instead of rendering <App> renders simple imape (white bg on light theme, black on dark. maybe with a logo).
super simple. yes its not gona be a pretty blur, just plain bg, but we will get rid of 2 deps.
and that blur we need only on rare occasions, only on few screens where user switches apps


@marcosrdz wdyt?

@limpbrains wdyt?

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by Overtorment** [comment]
LGTM to this PR, but i feel like we gotta do smth about it. 2 shitty deps for such smoll feature  is too much

**Comment by limpbrains** [comment]
https://www.npmjs.com/package/@react-native-community/blur works fine only on iOS, on android we glitchy.
Can't say anything about other deps really.

**Comment by marcosrdz** [comment]
That blur package is not the same thing. Its to blur components for styling it does not blur the app when a flag such as screenshot taken is activated

**Pull request merged**